### PR TITLE
Add a more  thorough explanation of the .isValid function in the docs

### PR DESCRIPTION
**Pull request by OmarOmeiri**

### CODE REVIEW

**Comment by iamkun** [comment]
Looks nice, can you submit it to the doc website, please?

https://github.com/dayjs/dayjs-website

File: docs/en/API-reference.md [hunk 1]
```diff
@@ -129,10 +129,39 @@ dayjs(dayjs('2019-01-25')) // passing a Dayjs object to a constructor will also
...
 Returns a `boolean` indicating whether the `Dayjs`'s date is valid.
 
-```js
-dayjs().isValid()
-```
-
+- Browser
+  - Non-strict check.
+
+    Only checks if the value can be `instanceof Date`.
```
Comment (iamkun) [inline docs/en/API-reference.md:135]: to check if it could be parsed to a Date time, is it easy to understand?

Comment (OmarOmeiri) [reply docs/en/API-reference.md:135]: Yes, it is very clear. Just to get it straight, you'd prefer if I change this: `...value can be instanceof Date.` to this: `checks if value could be parsed to a Date time`?

**Comment by iamkun** [comment]
close https://github.com/dayjs/dayjs-website/pull/71

**Pull request closed without merging**